x-pack/test_serverless/functional/test_suites/search/common_configs/config.group3.ts

```diff
@@ -12,9 +12,9 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
 
   return {
     ...baseTestConfig.getAll(),
-    testFiles: [require.resolve('../../common/visualizations/group2')],
+    testFiles: [require.resolve('../../common/visualizations/group3')],
     junit: {
-      reportName: 'Serverless Search Functional Tests - Common Group 2',
+      reportName: 'Serverless Search Functional Tests - Common Group 3',
```

Comment:
This text belongs to this config file and is independent of the loaded test files, so this change is correct. Good spot.

Reply:
I think I got it right [now](https://github.com/elastic/kibana/pull/180127/commits/af5ffa3897c2eee5dbfcfa2ce19cd4f1e0c4cf52) boss.